src/app/indiv-link/indiv-link.component.ts

```diff
@@ -17,10 +17,14 @@ export class IndivLinkComponent implements OnInit {
   regex: string;
   expand: boolean = false;
 
+  readonly highlightColor: string = 'rgb(253, 255, 71)'
+  nextBgColor: string = this.highlightColor;
```

Comment:
What do you think of having these fields instead:

originalDomColor: string
isHighlighted: bool

That way, the class is much more simple. We just need too maintain whether a boolean state instead of string.